### ISSUE TITLE
Inability to Import DataSets into "Own Space"

### ISSUE THREAD

**Issue by scottx611x** [refinery-platform/refinery-platform#2773]
* Specific code commit: 40deb27b2352af2a62d897e3763fc758e43271af
* Version of the web browser and OS: `Ubuntu 16.04 LTS Chrome`
* Environment where the error occurred : https://test.stemcellcommons.org/

### Steps to reproduce
Please list all the actions and the input data used:

1. Preview a DataSet on the Dashboard that allows for: "importing into your own space"
2. Click the `Import into own space` button

### Observed behavior
```

/#/preview/f548366c-1146-4bf6-9c42-f70ff488f89f:1 Mixed Content: The page at 'https://test.stemcellcommons.org/#/preview/f548366c-1146-4bf6-9c42-f70ff488f89f' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://test.stemcellcommons.org/data_set_manager/import/isa-tab-form/ajax/'. This request has been blocked; the content must be served over HTTPS.
angular.js:14110 TypeError: Cannot read property 'uuid' of undefined
    at dashboard.js:2181
    at angular.js:16606
    at m.$eval (angular.js:17913)
    at m.$digest (angular.js:17727)
    at m.$apply (angular.js:18021)
    at angular.js:18322
    at e (angular.js:6035)
    at angular.js:6314
(anonymous) @ angular.js:14110
(anonymous) @ angular.js:10633
(anonymous) @ angular.js:16614
$eval @ angular.js:17913
$digest @ angular.js:17727
$apply @ angular.js:18021
(anonymous) @ angular.js:18322
e @ angular.js:6035
(anonymous) @ angular.js:6314
setTimeout (async)
k.defer @ angular.js:6312
s @ angular.js:18321
$applyAsync @ angular.js:18053
l @ angular.js:11999
D @ angular.js:12211
e @ angular.js:12149
error (async)
(anonymous) @ angular.js:12152
n @ angular.js:11947
(anonymous) @ angular.js:11742
(anonymous) @ angular.js:16606
$eval @ angular.js:17913
$digest @ angular.js:17727
$apply @ angular.js:18021
(anonymous) @ angular.js:26495
dispatch @ jquery.js:4409
r.handle @ jquery.js:4095
XHR failed loading: GET "http://test.stemcellcommons.org/data_set_manager/import/isa-tab-form/ajax/".
angular.js:12185 XHR failed loading: POST "https://test.stemcellcommons.org/data_set_manager/import/take_ownership/".
```

### Expected behavior
* For a DataSet to be able to be imported into one's own space

**Comment by scottx611x** [comment]
Locally I'm seeing:
```
2018-05-21 12:58:17 ERROR    django.request:256 handle_uncaught_exception() - Internal Server Error: /data_set_manager/import/isa-tab-form/ajax/
Traceback (most recent call last):
  File "/home/vagrant/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/vagrant/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/vagrant/refinery/data_set_manager/views.py", line 225, in get
    os.unlink(temp_file_path)
OSError: [Errno 2] No such file or directory: '/vagrant/media/file_store/temp/isa_16410_959845_2_aG9KH0S.zip'
```

**Comment by scottx611x** [comment]
Closed by c74927404cdb5e88a473d7b70c8fb2cd7b312abf
